ledger-api/grpc-definitions/com/daml/ledger/api/v1/transfer.proto

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package com.daml.ledger.api.v1;
+
+import "com/daml/ledger/api/v1/event.proto";
+import "google/protobuf/timestamp.proto";
+
+option java_outer_classname = "TransactionOuterClass";
+option java_package = "com.daml.ledger.api.v1";
+option csharp_namespace = "Com.Daml.Ledger.Api.V1";
+
+// Complete view of an on-ledger transfer.
+message Transfer {
```

Comment:
I think, others suggested to have the initiator as well (https://github.com/digital-asset/daml/pull/15403#discussion_r1010265063). Maybe : `string initiator` ?

Reply:
What would be the initiator?

Reply:
The party that initiates the transfer-out/in

Reply:
wdyt of `submitter`? That’s what we use for command submissions and this seems morally the same.

Reply:
I could not find this submitter what you are referring to @cocreature , I mean it is there on the index-DB level, but that is not exposed on the Ledger API.
On the ledger API currently we have act_as for command submissions, and act_as for completions which is equivalent of the submitters (I think) you might refer to, but we do not expose them for daml transactions AFAIK.
Also for transfers we have the same act_as on submission.
Should we just stick to `act_as` so it is consistent?
@andreaslochbihler-da did you mean this information in your comment? There you talk about participant recognizing whether it was the initiator, but here we talk about submitting parties...does this serve that purpose?
On any rate: if we introduce parties here, we could follow the usual pattern on the Ledger API: we only expose parties which are requested, so this set of parties then would be filtered by the party set which is defined by the TransactionFilter.

Reply:
I meant it more as in "submitter is the terminology we use when talking about it" rather than there being an actual field called submitter.

Reply:
As for filtering, I'm not sure your analogy is right here. I think this is more like the controllers of an exercise which are included to everyone that sees the event.

Reply:
> @andreaslochbihler-da did you mean this information in your comment?

No. My understanding for user management is that each user is associated with a act_as and a read_as set to be used for this user's command submissions. If we want to extend this mechanism to transfers, then I wondered how we'd pick the single submitter from the set of act_as that comes from the user.

Reply:
@andreaslochbihler-da : on

> No. My understanding for user management is that each user is associated with a act_as and a read_as set to be used for this user's command submissions. If we want to extend this mechanism to transfers, then I wondered how we'd pick the single submitter from the set of act_as that comes from the user.

this is not how it works. see @cocreature 's comment 

> User management does not automatically pick the party. It still requires that you specify it explicitly and then the auth service checks that the party you specify is a subset of the parties you have claims for.
> 
> The JSON API does support automatically picking the party but you can still specify it manually. So I'd just error out if there is more than one actAs party there and require users to override it.

`act_as` and `read_as` must be specified by the Ledger API client, and then get checked against the rights granted to the user. There is one field that the Ledger API server fills out: that is the `application_id`, which is set to the `user_id` of the authenticated user.

Reply:
Great. Then there's no problem with supporting only a single submitter.

Reply:
@andreaslochbihler-da How is the initiator party helping to decide from which participant it was sent from? AFAIK the Transfer.command_id should signal precisely that (if present it was sent from that participant, if we follow the intent from Transaction.command_id).
Or if the command_id presence is not enough (not sure how completions work if we have multi-participant-hosted parties) how is knowing the submitter party helping more to make this decision?

Reply:
> How is the initiator party helping to decide from which participant it was sent from?

I don't think the initiator party helps with that.

Reply:
As discussed we introduce `submitter` for this purpose (singular, non filtered, always referring to the submitting party of the respective command, also adding this to TransferCommand)